core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -881,6 +896,12 @@ class CivilizationInfo {
             ?: return null
         if (unit.isGreatPerson()) {
             addNotification("A [${unit.name}] has been born in [${cityToAddTo.name}]!", placedUnit.getTile().position, unit.name)
+            for (unique in getMatchingUniques("Land units may cross [] after the first [] is earned")) {
```

Comment:
Uniques with tileFilters should be formatted "[tileFilter] tiles" for readability - also relevant to the "ending their turn" unique